autoload/nvimgdb.vim

```diff
@@ -226,6 +226,7 @@ endfunction
 
 function! s:DefKeymapVars()
   " Set global key maps
+  call s:DefKeymapVar("t:nvimgdb_key_until",      "g:nvimgdb_key_until",      "<c-u>" )
```

Comment:
I'd appreciate if the default keymap were something like `<f4>`, because `<c-u>` is page up in the default Vim.

Reply:
Is this what you suggest? I've never used f keys for anything and have the idiotic TB on my MacBook and don't know which fkeys are free.

Reply:
Yes, precisely. Anyone can still define whatever is most suitable for them in a personal vimrc. But by default, it'd better not overlap with Vim defaults.